gems/aws-sdk-s3/spec/object/download_file_spec.rb

```diff
@@ -221,17 +221,34 @@ module S3
         end
 
         it 'raises an error when checksum validation fails on multipart' do
-          client.stub_responses(:get_object, {body: 'body', checksum_sha1: 'invalid'})
+          client.stub_responses(:get_object, { body: 'body', checksum_sha1: 'invalid' })
 
           thread = double(value: nil)
-          allow(thread).to receive(:abort_on_exception=)
           expect(Thread).to receive(:new).and_yield.and_return(thread)
 
           expect do
             large_obj.download_file(path)
           end.to raise_error(Aws::Errors::ChecksumError)
         end
 
+        it 'does not interrupt the main thread when an exception occurs' do
```

Comment:
Probably not necessary because we're removing functionality here.

Reply:
Whoah. The `threat_report_on_exception` flag is cool. I was actually wondering on how I can go about removing the exception clutter left between specs. I found the PR when this was implemented: https://github.com/aws/aws-sdk-ruby/pull/2307

I want to say that it would be nice to keep it since the below spec has it: 
https://github.com/aws/aws-sdk-ruby/blob/ed4f82ecc9e141aafa2fd20bd24719db839510d3/gems/aws-sdk-s3/spec/object/multipart_copy_spec.rb#L371

Without this, this spec prints (7) long exception errors.

Also dropped the spec I wrote.